cdap-docs/admin-manual/source/operations/tx-maintenance.rst

```diff
@@ -0,0 +1,23 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+===============================
+Transaction Manager Maintenance
+===============================
+
+.. highlight:: console
+
+Transactions are marked as invalid when the changes made during the transaction cannot be rolled back. Transaction
+Manager has to store all invalid transactions so that they can be excluded from all future reads. This invalid 
```

Comment:
I am wondering if we should call it Transaction Service? That is what it is called in the UI: "transaction 
Service that maintains transaction states."